src/main/java/ch/njol/skript/expressions/ExprHiddenPlayers.java

```diff
@@ -0,0 +1,95 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import java.util.List;
+import java.util.ArrayList;
+import org.eclipse.jdt.annotation.Nullable;
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.util.coll.CollectionUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+
+@Name("Hidden Players")
+@Description({"The players hidden from a player that hidden using the <a href='effects.html#EffPlayerVisibility'>player visibility</a> effect."})
+@Examples({"message \"<light red>You are currently hiding: <light gray>%hidden players of the player%\""})
+@Since("INSERT VERSION")
+public class ExprHiddenPlayers extends SimpleExpression<Player> {
+
+	static {
+		Skript.registerExpression(ExprHiddenPlayers.class, Player.class, ExpressionType.PROPERTY,
+				"[(all [[of] the]|the)] hidden players [(of|for) %players%]",
+				"[(all [[of] the]|the)] players hidden [(from|for) %players%]",
+				"%players%'[s] [all] hidden players");
```

Comment:
`player's all hidden players` isn't grammatically correct, and i think that even `player's hidden players` sounds awkward so maybe don't have this at all?

Reply:
Yeah probably. I will remove it and add "by" to the 2nd pattern:
```autohotkey
[(all [[of] the]|the)] players hidden [(from|for|by) %players%]
```